### ISSUE TITLE
Can't find MyConnectionStrings. Config

### ISSUE THREAD

**Issue by MetSystem** [arichika/AspNet.Identity.Oracle#3]

**Comment by arichika** [comment]
By using own Data Source / ID / Password, Please create a file in the following format.  
  
```
<?xml version="1.0" encoding="utf-8"?>
<connectionStrings>
  <add name="DefaultConnection"
       connectionString="Data Source=OracleDb; User Id=scott;Password=tiger;"
       providerName="Oracle.DataAccess.Client" />
</connectionStrings>
```
And, Please save it as "MyConnectionStrings.config" in the same folder as the Web.config.  
  

If you use modified 'ApplicationDbContext.Create(string connectionName)',
You will need to change the name from "DefaultConnection".  

and for You. 
- Data Provider for .NET Developer's Guide
http://docs.oracle.com/cd/E51173_01/win.122/e17732/featConnecting.htm

**Comment by arichika** [comment]
@MetSystem  OK? https://github.com/arichika/AspNet.Identity.Oracle/issues/3#issuecomment-134117118

**Comment by MetSystem** [comment]
Thank you very much.

**Comment by cengizani** [comment]
Hi Arichika, Firstly, thank you so much for this project. It is good idea. I want to use your solution with our oracle db. I have completed register and login process. But when  I want to user administration and group management  then I have an ApplicationDbInitializer problem. So, when unblock code start with "public class ApplicationDbInitializer : " in IdentityConfig.cs and IdentityModels.cs " Database.SetInitializer<" code block then have an error 3 Message:"The type 'IdentitySample.Models.ApplicationDbContext' cannot be used as type parameter 'TContext' in the generic type or method 
'System.Data.Entity.DropCreateDatabaseIfModelChanges<TContext>'. 
There is no implicit reference conversion from 'IdentitySample.Models.ApplicationDbContext' to 'System.Data.Entity.DbContext'." 
Would you like the help me for this problem? Thanks again.

**Comment by arichika** [comment]
@cengizanil humm, I have understood your situation. Let me think about it.

**Comment by arichika** [comment]
@cengizani  humm, I have understood your situation. Let me think about it.

**Comment by cengizani** [comment]
Thank you very much I am waiting

**Comment by cengizani** [comment]
When Enable-Migrations from PM console then have a "No context type was found" message.
PM> Enable-Migrations
No context type was found in the assembly 'SampleWebSite'.

**Comment by cengizani** [comment]
I found something, apologizing is made;
In IdentityModels.cs ApplicationDbContext class inherit from OracleDatabase. 
 "public class ApplicationDbContext : OracleDatabase" 
in some examples on the Internet an inherit from IdentityDbContext with applicationuser
I wonder how it's done in the original MySql instance of info?
"public class ApplicationDbContext : IdentityDbContext<ApplicationUser>"
or
"public class ApplicationDbContext : IdentityDbContext<applicationuser, applicationuserclaim="">"

**Comment by arichika** [comment]
"AspNet.Identity.Oracle" has been designed so that it does not use any of Entity Framework.
So, I think that "IdentityDbContext" can not be used.

Be in this code there is a bug...
https://github.com/arichika/AspNet.Identity.Oracle/blob/master/Src/SampleWebSite/App_Start/IdentityConfig.cs#L136

"new RoleStore<IdentityRole>(param)"  requires OracleDatabase object.
https://github.com/arichika/AspNet.Identity.Oracle/blob/master/Src/AspNet.Identity.Oracle/RoleStore.cs#L46

The code might need  cast likes 
https://github.com/arichika/AspNet.Identity.Oracle/blob/master/Src/SampleWebSite/App_Start/IdentityConfig.cs#L29

Want you to create admin role for ApplicationDbInitializer unblock code? Right? @cengizani 

and I will create new issue ticket for this question.

**Comment by arichika** [comment]
Following is here #4

**Comment by cengizani** [comment]
I am controlling code for a bug
